test/services/cache/ExportsDeclarationBuilder.scala

```diff
@@ -159,7 +159,7 @@ trait ExportsDeclarationBuilder {
   def withBorderTransport(
     borderModeOfTransportCode: String = "",
     meansOfTransportOnDepartureType: String = "",
-    meansOfTransportOnDepartureIDNumber: Option[String] = None
+    meansOfTransportOnDepartureIDNumber: String = "1"
```

Comment:
Why not empty string?

Reply:
Count not be empty

Reply:
Yes, but `borderModeOfTransportCode` cannot be empty too and it contain empty string